src/PROPOSAL/PROPOSAL/methods.h

```diff
@@ -34,8 +34,11 @@
 #include <memory>
 #include <vector>
 #include <sys/stat.h>
+#ifdef _WIN32
```

Comment:
Why is this necessary?
a comment would be helpful

Reply:
unistd.h is a header that provides access to the API of the operating system, in this case to check paths (this header has also been used in older PROPOSAL versions).
However, unistd.h is by default not available for Windows. But for Visual Studio, io.h can be used as a replacement for unistd.h.

I am not 100% sure whether this fix works for all Windows systems, or only for those using Visual Studio. I should probably have a closer look at this. Nevertheless, I will add a comment to this for clarification.

Reply:
Note that this is actually not part of this PR, but part of the PR #305. I've just rebased this branch onto the branch of PR #305, so the changes are shown here.